screens/DemoScreen.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { View, Image, Text, Pressable, Platform, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import MaskedView from '@react-native-masked-view/masked-view'
+import EStyleSheet from 'react-native-extended-stylesheet';
+
+function DemoScreen({ navigation }) {
+    const backgroundCircle = require('../assets/background_blob.png');
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.headingContainer}>
+                <View style={styles.innerContainer}>
+                    <Text style={styles.headText}>Hey there!</Text>
+                    <Image source={require('../assets/seedling.png')} style={styles.seedling} />
+                </View>
+                <Text style={styles.text}>
+                    have an item you want to donate?
+                </Text>
+            </View>
+
+            <View style={styles.firstDemo}>
+                <Image source={require('../assets/demo1.png')} style={styles.numberFirst}/>
+                <Text style={styles.descriptionText}>tap the camera{"\n"}or photo upload{"\n"}button</Text>
+            </View>
+
+            <View style={styles.firstDemo}>
+                <Image source={require('../assets/demo_camera.png')} style={styles.demoCamera}/>
+                <Image source={require('../assets/demo_upload.png')} style={styles.demoUpload}/>
+            </View>
+
+            <View style={styles.secondDemo}>
+                <Image source={require('../assets/demo2.png')} style={styles.numberSecond}/>
+                <Text style={styles.descriptionText}>snap a photo of your item 📸</Text>
+            </View>
+
+            <View style={styles.thirdDemo}>
+                <Image source={require('../assets/demo3.png')} style={styles.numberThird}/>
+                <Text style={styles.descriptionText}>we’ll find donation centres{"\n"}that will accept your item!</Text>
+            </View>
+
+            <Image source={require('../assets/homescreen.png')} style={styles.homescreen}/>
+
+            <Image source={require('../assets/demo_girl.png')} style={styles.demoGirl} />
+            <TouchableOpacity onPress={() => navigation.navigate('WelcomeScreen')} style={styles.continueButton}>
+                <Text style={styles.continueText}>continue</Text>
+                <Image source={require('../assets/continue.png')} style={styles.continueImage}/>
+            </TouchableOpacity>
+        </View>
+    );
+}
+
+const styles = EStyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#fff',
```

Comment:
The background colour for the app is not pure white. Rebase this branch onto `main` to make this branch compatible with current development, and consider using the [`Screen`](https://github.com/ashenafee/STG/blob/main/components/Screen.tsx) component.

Reply:
Fixed